services/QuillLMS/app/views/pages/shared/_schools_and_districts_navbar.html.erb

```diff
@@ -0,0 +1,15 @@
+<% tabs = [
```

Comment:
You could abstract this structure into the NavigationHelper, to reduce the template LoC